packages/unenv-preset/package.json

```diff
@@ -20,7 +20,9 @@
 		"directory": "packages/unenv-preset"
 	},
 	"license": "MIT OR Apache-2.0",
-	"sideEffects": false,
+	"sideEffects": [
```

Comment:
Is this needed (there is a package.json in that folder)

Reply:
Vite didn't pick it up.

Reply:
Vite doesn't appear to pick that up.

Reply:
I think it is because, as far as this library is concerned `dist/runtime/polyfill` is not an entry-point to the package?
We have just a general export in the root of the package for `./*`.

Reply:
TIL `sideEffects` can be an array too. For unenv we use a package.json inside dist ([this](https://unpkg.com/browse/unenv@2.0.0-rc.14/dist/runtime/polyfill/package.json)) and it worked well.